### DISCUSSION TITLE
Hide "completed" iterations when grouping iterations

### DISCUSSION THREAD

**Discussion by Thalley** [community/community#11543] [Projects and Issues]
If using "group by iteration" in a view to get an overview of your backlog, current and upcoming iterations, it would be a nice feature if we could hide iterations older than the "current" iteration.

**Comment by nestor-sk** [comment]
Agree, this is a much-needed addition. Atm we need to manually exclude them

**Reply by nestor-sk** [reply]
I just saw that there is a better discussion opened that include this one https://github.com/github/feedback/discussions/10862
